.nvmrc

```diff
@@ -0,0 +1 @@
+16.14.0
```

Comment:
According to the [docs on nvm's repo](https://github.com/nvm-sh/nvm#nvmrc): 'The contents of a .nvmrc file must be the <version> (as described by nvm --help) followed by a newline.' and it looks like this doesn't have a newline.

Reply:
It doesn't appear to matter but i've fixed it anyway.